Jenkinsfile

```diff
@@ -203,7 +203,7 @@ pipeline {
                 pg_ctl -D /var/jenkins/${GIT_COMMIT}-${BUILD_NUMBER}/ -o '-p 5433' -l /var/jenkins/${GIT_COMMIT}-${BUILD_NUMBER}/events.log start; \
                 psql -h localhost -d postgres -p 5433 -U ${IROHA_POSTGRES_USER} --file=<(echo create database ${IROHA_POSTGRES_USER};)
               """
-              def testExitCode = sh(script: 'IROHA_POSTGRES_HOST=localhost IROHA_POSTGRES_PORT=5433 cmake --build build --target test', returnStatus: true)
+              def testExitCode = sh(script: """IROHA_POSTGRES_HOST=localhost IROHA_POSTGRES_PORT=5433 cd build && ctest --output-on-failure """, returnStatus: true)
               if (testExitCode != 0) {
                 currentBuild.currentResult = "UNSTABLE"
```

Comment:
Please, consider setting 'UNSTABLE' result using `currentBuild.result` property instead of `currentBuild.currentResult`. It probably became read-only in later Jenkins releases.